src/NHSD.BuyingCatalogue.Ordering.Api/Attributes/LimitAttribute.cs

```diff
@@ -0,0 +1,166 @@
+﻿using System;
```

Comment:
There's a lot of code in this class, would you mind creating some unit tests to provide some code coverage?

Reply:
Sure I can do that!